maintainers/maintainer-list.nix

```diff
@@ -14474,7 +14474,12 @@
   };
   onur-ozkan = {
     name = "Onur Ozkan";
-    email = "contact@onurozkan.dev";
+    email = "contact@onurozkan.dev";anirrudh = {
+    email = "anik597@gmail.com";
+    github = "anirrudh";
+    githubId = 6091755;
+    name = "Anirrudh Krishnan";
+  };
```

Comment:
It seems that these are accidental changes, can you look into this, @savedra1?

Reply:
Apologies! Have updated this now.